Adyen/Assets/Generated/LocalizationKey.swift

```diff
@@ -71,6 +71,10 @@ public struct LocalizationKey {
     public static let cardStoredMessage = LocalizationKey(key: "adyen.card.stored.message")
     /// Expires %@
     public static let cardStoredExpires = LocalizationKey(key: "adyen.card.stored.expires")
+    /// %@ isn't supported
+    public static let cardNumberItemUnsupportedBrand = LocalizationKey(key: "adyen.card.numberItem.unsupportedBrand")
+    /// The entered card brand isn't supported
+    public static let cardNumberItemUnknownBrand = LocalizationKey(key: "adyen.card.numberItem.unknownBrand")
```

Comment:
there is already `error.va.sf-cc-num.03` and `nfcScanner.error.card.unsupported` 🤔